sysutils/cdrtools/Portfile

```diff
@@ -1,98 +1,205 @@
+                        ${worksrcpath}/sformat/datio.c \
+                        ${worksrcpath}/smake/job.c \
+                        ${worksrcpath}/smake/make.c \
+                        ${worksrcpath}/sunpro/Make/bin/make/common/misc.cc \
+                        ${worksrcpath}/sunpro/Make/bin/make/common/read2.cc \
+                        ${worksrcpath}/TEMPLATES/Defaults.gcc \
+                        ${worksrcpath}/TEMPLATES/Defaults.clang \
+                        ${worksrcpath}/TEMPLATES/Defaults.xcc \
+                        ${worksrcpath}/ved/ved.h
 }
 
-use_configure       no
-
-# hangs the same way as smake itself with both llvm-gcc-4.2 and Xcode 4.1's clang
-# https://trac.macports.org/ticket/30310
-compiler.blacklist  llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
-
-configure.cflags-append \
-                    -DNO_SCANSTACK \
-                    -Wno-error=implicit-function-declaration
-
-build.cmd           smake
-build.args          CC=${configure.cc} \
-                    COPTX="${configure.cflags} [get_canonical_archflags cc]" \
-                    DEFOSINCDIRS=${prefix}/include \
-                    LDOPTX="${configure.ldflags} [get_canonical_archflags ld]" \
-                    LDPATH=
-
-# smake does not support -j
-# Set build.jobs to -1 to disable MacPorts from adding the -j flag
-build.jobs          -1
-
-post-build {
-    # The build script doesn't notice when a program fails to build, so
-    # we must verify it ourselves. See
-    # https://trac.macports.org/ticket/34823
-    set progs {btcflash cdda2wav cdrecord devdump isodebug isodump isoinfo isovfy mkisofs readcd scgcheck scgskeleton}
-    fs-traverse path ${worksrcpath} {
-        if {[file isfile ${path}] && [string match {*/OBJ/*} ${path}]} {
-            set file [file tail ${path}]
-            set prog_index [lsearch ${progs} ${file}]
-            if {${prog_index} != -1} {
-                set progs [lreplace ${progs} ${prog_index} ${prog_index}]
-            }
-        }
-    }
-    if {[llength ${progs}] > 0} {
-        ui_error "The following programs did not build: [join ${progs} {, }]"
-        return -code error "build failed"
-    }
+conflicts_build-append  cdrtools
+
+compiler.blacklist      llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
+
+configure.ldflags-append -lintl
+
+#
+# Note:  The current version of smake in the MacPorts repo no longer builds the star port
+#        properly.  One must use a modern version of gnumake (port:gmake) instead.
+#
+
+#
+# Once a newer version of smake is merged into the MacPorts Repo,
+# Replace "port:gmake" with "port:smake" as a build dependency
+# and change build.cmd to "smake" and delete the pre-destroot block
+#
+
+# depends_build-append    port:smake \
+                          port:gettext-runtime \
+                          port:libiconv
+
+depends_build-append    port:gmake \
+                        port:gettext-runtime \
+                        port:libiconv
```

Comment:
Since `libiconv` is already a lib dep, you don't need to declare a build dep for it.

Reply:
> Since `libiconv` is already a lib dep, you don't need to declare a build dep for it.

it looks like this was the only thing you missed, with your latest push.

Reply:
Are you sure?  I see "localization" of various files during both the build and destroot stages.  

I soon discovered that I needed to add the "-lintl" flag to "configure.ldflags-append".  Otherwise the build would fail on Big Sur and above.

Are you telling me that I just need the "-intl" flag?  And I do not need the "libiconv" as a build dependency?  Is that right?

Reply:
> Are you sure? I see "localization" of various files during both the build and destroot stages.
> 
> I soon discovered that I needed to add the "-lintl" flag to "configure.ldflags-append". Otherwise the build would fail on Big Sur and above.
> 
> Are you telling me that I just need the "-intl" flag? And I do not need the "libiconv" as a build dependency? Is that right?

In short, all lib deps are also build deps. So if it's declared as a lib dep, it will also be pulled in for the build. (You'll still need your linker flag though, obviously.)